EventBrite/Models/EventBriteItem.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Etch.OrchardCore.Fields.EventBrite.Models.Dto;
+using System;
+
+namespace Etch.OrchardCore.Fields.EventBrite.Models
+{
+    public class EventBriteItem
+    {
+        public EventBriteItem()
+        {
+        }
+
+        public EventBriteItem(EventBriteEventDto eventDto)
+        {
+            UpdateFromEventDto(eventDto);
+        }
+
+        public EventBriteItem(EventBriteEventDto eventDto, EventBriteVenueDto venueDto)
```

Comment:
As per a previous comment, I'd make `venueDto` default to `null` (shown below). Then where the address is set, have a null check.

```
public EventBriteItem(EventBriteEventDto eventDto, EventBriteVenueDto venueDto = null) 
{
...
```

Reply:
Had a chat with PK, decided to make both parameters nullable, remove the empty constructor and only update fields depending on which dtos we've been passed